docs/howto/ayx-api-example-remove-all-users.md

```diff
@@ -0,0 +1,96 @@
+**Overview**
+
+This guide is for the use case of any workspace admin who needs to clear out a workspace except themselves. This is especially useful for universities using Alteryx Analytics Cloud when they have new classes of students in their workspace each semester.
+
+**Guide**
+---------
+
+This guide assumes limited knowledge of Python, but some research and debugging may be required by the user to set up Python and Pip Steps:
+
+1. Make sure you have Python installed by running both of the following in your terminal (whichever works, you'll use the same prefix for later commands):
+   1. `python --version`
+   2. `python3 --version`
+
+2. Install the library to make API calls in Python "Requests" using one of the following commands depending on whether you have "python" or "python3" installed:
+   1. `pip install requests`
+   2. `python -m pip install requests`
+   3. `python3 -m pip install requests`
+   4. If none of these work, you may need to install pip
+
+3. Edit "removeAllUsersExceptExcluded.py" to match your workspace and needs
+   1. Edit the `bearer_token` variable to be your static bearer token from the "user preferences" then Static API access tokens pages (link). These tokens will be deprecated soon so eventually this script will need to be updated to use the oauth 2.0 tokens
+   2. Edit the `excluded_emails` array to include all the emails of users you don't want to be removed. If it's just you, make sure there are no commas and just your email in the brackets: `[ ]`.
+   3. Save your changes
+
+4. Run the python script using one of the following commands depending on whether you have "Python" or "Python3" installed
+   1. `python removeAllUsersExceptExcluded.py`
+   2. `python3 removeAllUsersExceptExcluded.py`
+
+**The Script**
+---------
+
+```python
+import requests
+
+# Variables to adjust based on the environment
+base_url = "us1.alteryxcloud.com"
+bearer_token = "YOUR_STATIC_BEARER_TOKEN_HERE"
+excluded_emails = ["user1_to_exclude@example.com", "user2_to_exclude@example.com"]  # Comma-seperated list of emails of users who should not be removed
+
+# Headers
+headers = {
+    "Authorization": f"Bearer {bearer_token}",
+    "Accept": "application/json"
+}
+
+# Function to make a GET request
+def make_get_request(endpoint):
+    # Complete URL
+    url = base_url + endpoint
```

Comment:
While there's nothing strictly wrong with this the way you're currently using it, it might be a good idea to use `urllib.parse.urljoin` to build the URL here